contributing-guides/style-guide.md

```diff
@@ -160,7 +160,7 @@ Use backticks on the following:
 
 ## Imperative Mood
 
-Example descriptions have to be phrased in imperative mood.  
+Example descriptions have to be phrased in an imperative mood.  
```

Comment:
IMHO this `an` isn't correct
```suggestion
Example descriptions have to be phrased in imperative mood.  
```

Reply:
I thought of using an article, I guess the current one is fine then.